libcontainer/container_linux.go

```diff
@@ -465,6 +465,7 @@ func (c *linuxContainer) commandTemplate(p *Process, childPipe *os.File) (*exec.
 	if p.ConsoleSocket != nil {
 		cmd.ExtraFiles = append(cmd.ExtraFiles, p.ConsoleSocket)
 		cmd.Env = append(cmd.Env,
+			fmt.Sprintf("GOMAXPROCS=%s", os.Getenv("GOMAXPROCS")),
```

Comment:
should this apply to only console socket ?, also we should check if GOMAXPROCS is indeed there before setting the env.

Reply:
oops